packages/vkui/src/components/AppRoot/helpers.ts

```diff
@@ -84,3 +85,31 @@ export const setSafeAreaInsets = (
     }
   };
 };
+
+export function getUserSelectModeClassName({
+  userSelectMode,
+  isWebView,
+  hasPointer,
+}: {
+  userSelectMode: AppRootUserSelectMode;
+  isWebView: boolean;
+  hasPointer: boolean | undefined;
+}): string | null {
+  switch (userSelectMode) {
+    case 'disabled-in-webview': {
+      return isWebView ? styles['AppRoot--user-select-none'] : null;
+    }
+    case 'enabled-with-pointer': {
+      return hasPointer === undefined
+        ? styles['AppRoot--pointer-none']
+        : !hasPointer
+        ? styles['AppRoot--user-select-none']
+        : null;
```

Comment:
```suggestion
      return hasPointer === undefined
        ? styles['AppRoot--pointer-none']
        : hasPointer ? null : styles['AppRoot--user-select-none'];
```

Мейби не инвертировать булик, а поменять местами значения?

Reply:
Не уверен, что стало лучше, но это натолкнуло на мысль написать иначе.

Изначально я взял условие из AppRoot, но сейчас переделал, кажется, что стало попонятнее: ⚡ https://github.com/VKCOM/VKUI/pull/7005/commits/ecb2e7196f276350cc0d37b9f88dc8d7144aebce